README.md

```diff
@@ -21,6 +21,10 @@ _The Taplytics team is available 24/7 to answer any questions you have. Just ema
 
 ## Changelog
 
+**[1.26.0](https://github.com/taplytics/Taplytics-Android-SDK/releases/tag/1.26.0)**
+
+1. Added support for opt-in/opt-out for compliance with GDPR.
```

Comment:
Think you forgot one of the bug fixes we did (see the QA build I sent yesterday)

Reply:
I think @VicV wanted to exclude the bug since we don't know if we actually fixed that bug.